mava/components/jax/building/distributor.py

```diff
@@ -17,9 +17,11 @@
 from dataclasses import dataclass
 from typing import Callable, List, Optional, Union
 
+import launchpad as lp
+
 from mava.components.jax import Component
 from mava.core_jax import SystemBuilder
-from mava.systems.jax.launcher import Launcher, NodeType
+from mava.systems.jax.launcher import Launcher, NodeType, copy_store
```

Comment:
Should we rather have `copy_store` in a utils file somewhere? Might be useful elsewhere. Just feels a little weird to have it belong in the launcher.

I would even prefer it being with the builder.

Reply:
Agreed :+1: I updated the code as suggested.